hardware_enablement/release-notes/kmm-release-notes.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="kmm-release-notes-v2-2"]
+= Kernel Module Management Operator release notes
+include::_attributes/common-attributes.adoc[]
+:context: kmm-release-notes-v2-2
+
+toc::[]
+
+[id="kmm-2-2"]
+== Release notes for Kernel Module Management Operator 2.2
+
```

Comment:
In the future, you might consider including the errata link, like how OCP does. For example: https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#ocp-4-17-8_release-notes

and maybe even a brief intro like that.